### ISSUE TITLE
DB, Calendar view: Record about rejected sick leave appears in [dbo].[SickLeaveCancellations]

### ISSUE THREAD

**Issue by KuchaLeni** [SoftwareCountry/arcadian-assistant#611]
1. Create sick leave
2. Approve sick leave from step 1 by supervisor
3. Reject approved sick leave

**Result:**
Record about rejected sick leave appears in [dbo].[SickLeaveCancellations]

**Expected result:**
Record about rejected sick leave appears in [dbo].[SickLeaveRejects]

**Comment by sponkrashin** [comment]
@antonm76 , the same question about request. What do you send to server in that case?

**Comment by antonm76** [comment]
@YaroTs could you please answer @sponkrashin ?

**Comment by YaroTs** [comment]
@sponkrashin 

```
PUT  /employees/2237/events/Sickleave_23
```

```
{
    "calendarEventId": "Sickleave_23",
    "type": "Sickleave",
    "dates": {
        "startDate": "04/15/2019",
        "endDate": "04/18/2019",
        "startWorkingHour": 0,
        "finishWorkingHour": 8
    },
    "status": "Cancelled"
}
```

**Comment by sponkrashin** [comment]
This is not correct. `Cancelled` is when employee cancels it's own event. When event is rejected by supervisor, `Rejected` status should be sent.

**Comment by antonm76** [comment]
@KuchaLeni @sponkrashin  , it is not possible to Reject already approved sick leave, it is only possible to Cancel it. Supervisor could Reject sick leave the very first time it appears, before it was approved by him or another supervisor.

**Comment by KuchaLeni** [comment]
Supervisor has the possibility to reject already approved sick leave on employee's profile. 
![image](https://user-images.githubusercontent.com/28759011/56142687-c6bd9080-5fa7-11e9-9425-9f1a69535625.png)

**Comment by sponkrashin** [comment]
Ok, we are tangled in terms. I'll ask @santee.

**Comment by antonm76** [comment]
@KuchaLeni "Discard" in this particular case means "Cancelled" status of an event. It was designed this way originally, but I'm not 100% sure it is correct.

**Comment by antonm76** [comment]
![Evernote Snapshot 20190204 185029](https://user-images.githubusercontent.com/6681817/56144079-611ed380-5faa-11e9-9516-dd7c4b0023ef.png)

**Comment by antonm76** [comment]
According to the table of states I attached, approved events can only be cancelled.

**Comment by sponkrashin** [comment]
@santee has just said that it doesn't really matter for now. He said it another way, not so politeness, but his words had such meaning. So let's postpone this and keep it as is now.
